src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -25,22 +26,26 @@ namespace Microsoft.Health.Fhir.Core.Features.Conformance
     public sealed class SystemConformanceProvider
         : ConformanceProviderBase, IConfiguredConformanceProvider, INotificationHandler<RebuildCapabilityStatement>, IAsyncDisposable
     {
+        private static SemaphoreSlim _defaultCapabilitySemaphore = new SemaphoreSlim(1, 1);
+        private static SemaphoreSlim _metadataSemaphore = new SemaphoreSlim(1, 1);
```

Comment:
Should we try making these not static? From our discussion it shouldn't matter, but I'm currious if it would change anything.

Reply:
I'm going to change that, and also add some log that I had an idea this morning.